fast64_internal/oot/oot_collision.py

```diff
@@ -347,28 +347,32 @@ def ootWaterBoxToC(waterBox):
 def ootCameraDataToC(camData):
 	posC = CData()
 	camC = CData()
+	hasPosData = exportPosData = False
```

Comment:
```suggestion
	exportPosData = False
```
I think setting `hasPosData` here is basically useless?